Android/src/org/droidplanner/android/utils/prefs/DroidPlannerPrefs.java

```diff
@@ -694,4 +698,18 @@ public void setVehicleDefaultSpeed(float speedInMetersPerSecond) {
         prefs.edit().putFloat(PREF_VEHICLE_DEFAULT_SPEED, speedInMetersPerSecond).apply();
         lbm.sendBroadcast(new Intent(PREF_VEHICLE_DEFAULT_SPEED));
     }
+
+    /**  default follow mode
+     * @return FollowType
+     */
+    public FollowType getLastKnownFollowType() {
+        String followTypeString = prefs.getString(PREF_LAST_KNOWN_FOLLOW_MODE, DEFAULT_FOLLOW_TYPE);
+        FollowType followType = FollowType.valueOf(followTypeString);
```

Comment:
Following from the comment above, the second argument in the `getString(...)` method should be updated to `DEFAULT_FOLLOW_TYPE.name()`.
In addition, you should check the return value from `FollowType.valueOf(followTypeString)`. If that value is `null`, then return the `DEFAULT_FOLLOW_TYPE` instead.